internal/service/anycast/model_proto_anycast_config.go

```diff
@@ -0,0 +1,189 @@
+package anycast
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/int64planmodifier"
+
+	"github.com/hashicorp/terraform-plugin-framework-timetypes/timetypes"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/infobloxopen/bloxone-go-client/anycast"
+
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/flex"
+)
+
+type ProtoAnycastConfigModel struct {
+	AccountId          types.Int64       `tfsdk:"account_id"`
+	AnycastIpAddress   types.String      `tfsdk:"anycast_ip_address"`
+	AnycastIpv6Address types.String      `tfsdk:"anycast_ipv6_address"`
+	CreatedAt          timetypes.RFC3339 `tfsdk:"created_at"`
+	Description        types.String      `tfsdk:"description"`
+	Fields             types.Object      `tfsdk:"fields"`
```

Comment:
I don't think `Fields` should be part of the model here, It's supposed to be useful for API, not terraform. How does this work in terraform as in examples?

Reply:
Fields are not used